network-primitives/scripts/create_bootstrap_node_compose_file.sh

```diff
@@ -20,6 +21,15 @@ services:
       - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
       - /etc/os-release:/host/etc/os-release:ro
 
+  dsn-bootstrap-node:
+    image: ghcr.io/\${NODE_ORG}/bootstrap-node:\${NODE_TAG}
+    volumes:
+      - dsn_boot_node:/var/subspace:rw
```

Comment:
Are you sure it stores something there? It shouldn't unless path is explicitly provided.